### PR TITLE
EAGLE-85 : Fix unit test failure.

### PR DESCRIPTION
**Pull request by RalphSu**
Fix the failed test cases.

1. avoid hive sql in test case using reserved words which fails the hive parser. https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL

2. Mark not-runnable tests (like external dependency, not well formatted scala test) as ignored.

3. Fix a couple of failed test cases in hbase audit log parser.

### CODE REVIEW

File: eagle-security/eagle-security-hive/src/test/java/org/apache/eagle/security/hive/ql/TestParser.java [hunk 1]
```diff
@@ -248,9 +249,11 @@ public void testDropTable() throws Exception {
...
...stamp, cln, siid, ciid, sellerid ) b on a.uid = b.uid and a.site_id = b.site_id and a.page_id = b.page_id and coalesce(a.curprice, 'null') = coalesce(b.curprice, 'null') and coalesce(a.itm, 'null') = coalesce(b.itm, 'null') and coalesce(a.itmcond, 'null') = coalesce(b.itmcond, 'null') and coalesce(trim(reflect(\"java.net.URLDecoder\", \"decode\",regexp_replace(a.itmtitle,\"\\\\+\",\" \"),\"utf-8\")),'null') = coalesce(trim(b.itmtitle),'null') and coalesce(a.l1, 'null') = coalesce(b.l1, 'null') and coalesce(a.l2,'null') = coalesce(b.l2,'null') and coalesce(a.leaf,'null') = coalesce(b.leaf,'null') and coalesce(a.meta,'null') = coalesce(b.meta,'null') and coalesce(a.st,'null') = coalesce(b.st,'null') and coalesce(a.dc,'null') = coalesce(b.dc,'null') and coalesce(reflect(\"java.net.URLDecoder\", \"decode\",a.tr,\"utf-8\"),'null') = coalesce(b.tr,'null') and a.eventtimestamp = b.eventtimestamp and coalesce(a.cln,'null') = coalesce(b.cln,'null') and coalesce(a.siid,'null') = coalesce(b.siid,'null') and coalesce(a.ciid,'null') = coalesce(b.ciid,'null') and coalesce(a.sellerid, 'null') = coalesce(b.sellerid, 'null') where b.uid is null distribute by a.uid sort by a.uid , a.eventtimestamp";
```
Comment (haoch) [inline eagle-security/eagle-security-hive/src/test/java/org/apache/eagle/security/hive/ql/TestParser.java]: Remove the sensitive table name

**Comment by haoch** [comment]
There are some sensitive table/field name in the unit test case sql, please rename/remove.

**Comment by RalphSu** [comment]
Updated.

**Comment by haoch** [comment]
Thanks for the fix but seems not completely clean yet :-)

**Comment by haoch** [comment]
https://github.com/RalphSu/incubator-eagle/blob/master/eagle-security/eagle-security-hive/src/test/java/org/apache/eagle/security/hive/ql/TestParser.java#L126

https://github.com/RalphSu/incubator-eagle/blob/master/eagle-security/eagle-security-hive/src/test/java/org/apache/eagle/security/hive/ql/TestParser.java#L153

https://github.com/RalphSu/incubator-eagle/blob/master/eagle-security/eagle-security-hive/src/test/java/org/apache/eagle/security/hive/ql/TestParser.java#L173

https://github.com/RalphSu/incubator-eagle/blob/master/eagle-security/eagle-security-hive/src/test/java/org/apache/eagle/security/hive/ql/TestParser.java#L262

**Comment by RalphSu** [comment]
i don't see why test_lateral_view_shengli/bm2_item4_5 are sensitive?

**Comment by RalphSu** [comment]
Make them as more meaningless ones t1 t2....

**Comment by haoch** [comment]
`shengli` should be someone's internal account name :-)

**Comment by haoch** [comment]
LGTM. Thanks very much for the contribution! @RalphSu

**Pull request closed without merging**